src/gpu/AssemblerAlign-gpu.cpp

```diff
@@ -124,8 +124,8 @@ void Assembler::computeAlignmentsGpu(
     cout << timestamp << "Creating alignment table." << endl;
     computeAlignmentTable();
 
-    cout << timestamp << "Shutting down processors." << endl;
-    shasta_shutdownProcessors(nDevices);
+    //cout << timestamp << "Shutting down processors." << endl;
+    //shasta_shutdownProcessors(nDevices);
```

Comment:
Why are we not shutting down the processors?

Reply:
They are now shut down later on (after disjoint set computation).